server/schedule/labeler/labeler.go

```diff
@@ -122,9 +178,26 @@ func (l *RegionLabeler) GetLabelRules(ids []string) ([]*LabelRule, error) {
 
 // GetLabelRule returns the Rule with the same ID.
 func (l *RegionLabeler) GetLabelRule(id string) *LabelRule {
-	l.RLock()
-	defer l.RUnlock()
-	return l.labelRules[id]
+	return l.getAndCheckRule(id, time.Now())
```

Comment:
Can we reduce the passing of the `now` parameter, which currently seems unnecessary?

Reply:
sorry, we can't. since we should check if there exist expired labels before return the current rule.